includes/class-image-optimisation.php

```
@@ -657,11 +657,75 @@ private function get_all_preload_data(): array {
+						$lcp_url = $transient;
+						break;
+					}
+				}
+			}
+
+			if ( empty( $lcp_url ) ) {
+				return array();
+			}
+
+			return array( $this->prepare_preload_item( $lcp_url ) );
+		}
```

PHPMD (2.15.0)

[warning] The method get_auto_lcp_preload_data() has a Cyclomatic Complexity of 13. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)